addons/spreadsheet/static/src/pivot/pivot_time_adapters.js

```diff
@@ -190,7 +200,10 @@ const quarterAdapter = {
     },
     formatValue(normalizedValue, locale) {
         const [quarter, year] = normalizedValue.split("/");
-        return sprintf(_t("Q%(quarter)s %(year)s"), { quarter, year });
```

Comment:
I don't agree with this change at all. Everywhere in Odoo it says "Q3 2023". Our formula use "Q3 2023". I do'nt think we should display "2023 Q3", that just seems strange. And very error prone if we the accouting function use "Q3 2023" but the pivot are displayed as "2023 Q3". 

If we really want sorting to work on quarters, we should handle a real format "qq yyyy", like we have for the months. 

(same remarks for the week)

Sorting part of a pivot doesn't look to be very much in demand anyway, it's bugged and impossible to do atm 😛 https://www.odoo.com/web#menu_id=4720&cids=1&action=333&active_id=2328&model=project.task&view_type=form&id=3581689

Reply:
Probably not related as it's already present but any translator could decide on their own to switch the terms to fit the customary representation of the dates in their language. This could then effectively break the sorting?